README.md

```diff
@@ -16,12 +14,50 @@
 
-In this lesson, we're going to learn the syntax and basic vocabulary of regular expressions. We'll start simple and build from there. A great place to head for Regex testing and practice is [Rubular](http://rubular.com/) - it allows you to build and test regular expressions against text that you define.
+In this lesson, we're going to learn the syntax and basic vocabulary of regular expressions. We'll start simple and build from there. A great place to head for Regex testing and practice is [Rubular](http://rubular.com/) - it allows you to build and test regular expressions against text that you define. In a separate window, open up Rubular. In the text box entitled "Your Test String", paste in the following monologue from Shakespeare's *The Merchant of Venice*:
 
+>If to do were as easy as to know what were good to do, chapels had been churches and poor men's cottages princes' palaces. It is a good divine that follows his own instructions: I can easier teach twenty what were good to be done, than be one of the twenty to follow mine own teaching. The brain may devise laws for the blood, but a hot temper leaps o'er a cold decree: such a hare is madness the youth, to skip o'er the meshes of good counsel the cripple. But this reasoning is not in the fashion to choose me a husband. O me, the word 'choose!' I may neither choose whom I would nor refuse whom I dislike; so is the will of a living daughter curbed by the will of a dead father. Is it not hard, Nerissa, that I cannot choose one nor refuse none?
+
+Your window should look like this:
+![rubular setup](http://curriculum-content.s3.amazonaws.com/web-development/Regex/rubular_setup.png)
 
 
 ## Writing Regular Expressions
 
-### Part 1: Do Some Stuff
+In Ruby, regular expressions are always written between forward slashes:
+`/your regex/`. In Rubular, you can see that these slashes have already been written for you.
+
+### Simple Text Matching
+
+Let's start with the simplest text matching. Add the following regex in rubular:
+```
+/twenty/
+```
+
+![twenty regex](https://curriculum-content.s3.amazonaws.com/web-development/Regex/twenty.png)
+
+Notice that the pattern matches the two instances of "twenty" in the passage. Writing a series of letters or numbers in your regular expression will result in a search for exact matches of this pattern anywhere in the string. 
+
+### Metacharacters
+The real beauty of Regular Expressions is revealed in its use of metacharacters. Metacharacters allow you to use a pre-defined shorthand to match specific characters. For example, `\d` will match any digit in your text, and `\w` will match any word character (non-digit, non-punctuation). The 'Regex Quick Reference' at the bottom of Rubular shows metacharacters and patterns that you can use. Play around with these a little. Use `\W` (notice uppercasing) to match the non-word characters in your text.
```

Comment:
`\w` matches anything from this character class `[a-zA-Z0-9_]`, so letters, number, and underscore